src/Protocol/DFRN.php

```diff
@@ -1313,19 +1359,19 @@ public static function deliver($owner,$contact,$atom, $dissolve = false) {
 			return -9; // timed out
 		}
 
-		if (($curl_stat == 503) && (stristr($a->get_curl_headers(),'retry-after'))) {
+		if (($curl_stat == 503) && (stristr($a->get_curl_headers(), 'retry-after'))) {
 			return -10;
 		}
 
-		if (strpos($xml,'<?xml') === false) {
+		if (strpos($xml, '<?xml') === false) {
 			logger('dfrn_deliver: phase 2: no valid XML returned');
 			logger('dfrn_deliver: phase 2: returned XML: ' . $xml, LOGGER_DATA);
 			return 3;
 		}
 
 		if ($contact['term-date'] > NULL_DATE) {
 			logger("dfrn_deliver: $url back from the dead - removing mark for death");
-			require_once('include/Contact.php');
+			include_once 'include/Contact.php';
```

Comment:
Why this change?

Reply:
this was a suggestion from phpcs, it first mentioned that require_once is a statement not a function. After removing the parenthesis, it then gave the suggestion that since it was a conditional include to use include_once instead of require_once.